### PR TITLE
Remove margin bottom from register fields

### PR DESCRIPTION
**Pull request by adeelkhan**
Screen Shot 2021-02-01 at 1 27 08 PM

### CODE REVIEW

File: src/register/RegistrationPage.jsx [hunk 1]
```diff
@@ -403,7 +403,7 @@ class RegistrationPage extends React.Component {
                 <ValidationFormGroup
                   for={field.name}
                   key={field.name}
-                  className="pt-10 mb-20 small"
+                  className="pt-10 mb-0 small"
```
Comment (waheedahmed) [inline src/register/RegistrationPage.jsx:406]: No need to add `mb-0` class, we need to keep `1rem` default `margin-bottom` set on `.form-group`.

**Review by waheedahmed — COMMENTED** [review]
Remove `mb-0`, we need the default `margin-bottom` set on `.form-group`

File: src/register/RegistrationPage.jsx [hunk 2]
```diff
@@ -683,7 +679,7 @@ class RegistrationPage extends React.Component {
                 { this.addExtraRequiredFields() }
                 <ValidationFormGroup
                   for="optional"
-                  className="custom-control pt-10 mb-0"
```
Comment (waheedahmed) [inline src/register/RegistrationPage.jsx:686]: This shouldn’t be removed. We need zero bottom margin on this.

Comment (adeelkhan) [reply src/register/RegistrationPage.jsx]: done

File: src/register/RegistrationPage.jsx (hunk 1 above)
Comment (adeelkhan) [reply src/register/RegistrationPage.jsx]: done

**Review by waheedahmed — APPROVED** [review]

**Pull request merged**